css/style.css

```diff
@@ -81,17 +65,373 @@ ul {
+    padding: 0;
+    grid-column: 1/4;
+    grid-row: 2;
+    align-self: flex-start;
+    padding-top: 77px;
+  }
+
+  #title {
+    flex-direction: row;
+    gap: 24px;
+    grid-column: 1/4;
+    grid-row: 1/2;
+    align-self: flex-start;
+  }
+
+  #workPlaceholder {
+    flex-grow: 1;
+    margin-right: auto;
+    background-image: url("../icons/Vectortitle.svg");
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: 100%;
+    background-origin: content-box;
+  }
+
+  .placeholder {
+    background-image: url("../icons/ImgPlaceholder2.svg");
+    padding-bottom: 42%;
+    width: 100%;
+  }
+
+  #multi-postBlock {
+    padding: 0;
+  }
+
+  #recent-works {
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: 0.1fr 0.3fr 0.3fr 0.3fr;
+    margin: 0;
+    padding: 130px 141px;
+    gap: 37px 24px;
+  }
+
+  #cardDesc {
+    font-size: 17px;
+    line-height: 141%;
+  }
+
+  #cardWork {
+    display: none;
+  }
+
+  #deskCardWork {
+    align-items: flex-start;
+    justify-content: flex-end;
+  }
+
+  #deskCardWork1 {
+    justify-content: flex-end;
+    background-image: url("../icons/deskCardWork1.svg");
+    background-size: cover;
+    background-repeat: no-repeat;
+  }
+
+  #cardElements1 {
+    background-image: url("../icons/deskCardWork2.svg");
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-origin: border-box;
+    gap: 24px;
+    padding: 178px 16px 16px;
+    flex: 1;
+    justify-content: flex-end;
+  }
+
+  #cardElements2 {
+    background-image: url("../icons/deskCardWork3.svg");
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-origin: border-box;
+    gap: 24px;
+    padding: 221px 16px 16px;
+    flex: 1;
+  }
+
+  #cardElements3 {
+    background-image: linear-gradient(180.45deg, rgba(38, 38, 38, 0) 0.75%, rgba(38, 38, 38, 0.9) 84.18%), url("../icons/deskCardWork1.svg");
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-origin: border-box;
+    gap: 24px;
+    padding: 178px 16px 16px;
+    flex: 1;
+    justify-content: flex-end;
+  }
+
+  #aboutMe {
+    padding: 130px 0;
+    width: 45%;
+  }
+
+  #amHolder {
+    padding: 0 142px;
+    background: url("../icons/deskAmBackGround.svg") right no-repeat, url("../icons/amVector.svg") bottom no-repeat;
+    background-repeat: no-repeat;
+    background-size: 50%, 100%;
+    background-origin: border-box, content-box;
+  }
+
+  #amPlaceHolder {
+    padding-top: 9px;
+  }
+
+  #amTitle {
+    line-height: 113%;
+  }
+
+  #amDesc {
+    font-size: 20px;
+    line-height: 160%;
+  }
+
+  #skills {
+    flex-direction: row;
+    align-items: baseline;
+    justify-content: center;
+    gap: 152px;
+    padding: 130px 142px;
+  }
+
+  #formContainer {
+    flex-direction: row;
+    gap: 24px;
+    padding: 130px 142px 154px 135px;
+  }
+
+  #contactDescription {
+    width: 31.6%;
+    font-size: 40px;
+    line-height: 130%;
+  }
```

Comment:
- [ ] Kindly correct the width of the contact description t be similar to the Figma design. 👇🏽 

| Figma design | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/213600679-7b985aa4-9ce2-4a5c-acf0-72c03c68728a.png) | ![image](https://user-images.githubusercontent.com/58642949/213600736-d22b6e2d-a8db-4c81-a37b-3b637566320e.png) |